dohq_artifactory/admin.py

```diff
@@ -93,9 +94,14 @@ def _create_and_update(self):
             headers={"Content-Type": "application/json"},
             auth=self._auth,
         )
-        r.raise_for_status()
-        rest_delay()
-        self.read()
+        try:
+            r.raise_for_status()
+        except requests.exceptions.HTTPError as e:
+            print("Response text: {}".format(e.resonse.text))
+            raise SystemExit(e)
```

Comment:
Let's introduce our own `Exception` - `ArtifactoryError`

Reply:
Do we want to introduce a different `ArtifactoryError`, or just reuse `ArtifactoryException`?

Reply:
We can reuse this one, if it fits us.